Puzzlepart/did
# Custom dev hbs template which clears storage on load

### Your checklist for this pull request
- [X] Make sure [CHANGELOG.md](https://github.com/Puzzlepart/did/blob/dev/CHANGELOG.md) is update if applicable

### Review checklist

### Description
Created a custom hbs-template (`dev_template.hbs`) with a simple script that  
if you're running on localhost and have localStorage data, clears local- and sessionStorage, and sets `sessionStorage.debug="true"`.  
This ensures you're debugging a fresh session, but also lets you persist any data you create in that session.

### How to test
1. Check out locally with [gh](https://github.com/cli/cli).
2. Ensure you have `NODE_ENV=development` in your `.env`-file.
3. Run `npm run watch`.
4. ignore some events in your time sheet, or some other operation that introduces data in local- or sessionStorage.
5. stop the server.
6. relaunch the server with `npm run watch` .
7. observe that `localStorage` and `sessionStorage` does not contain any did timesheet-data-related entries.

### Related issues
Closes #715
